app/domain/authentication/authn_jwt/vendor_configurations/configuration_factory.rb

```diff
@@ -7,12 +7,12 @@ class ConfigurationFactory
           "authn-jwt" => ConfigurationJWTGenericVendor
         }
 
-        def create_jwt_configuration(authenticator_name)
-          unless AUTHENTICATORS[authenticator_name]
-            raise Errors::Authentication::AuthnJwt::UnsupportedAuthenticator, authenticator_name
+        def create_jwt_configuration(authenticator_input)
+          unless AUTHENTICATORS[authenticator_input.authenticator_name]
```

Comment:
i would add here `authenticator_name = authenticator_input.authenticator_name` and then use it in the other usages

Reply:
Done